content/applications/finance/accounting/receivables/customer_payments/batch.rst

```diff
@@ -2,34 +2,20 @@
 Batch payments: Batch deposits (checks, cash etc.)
 ==================================================
```

Comment:
New title proposition: "Batch payments by bank deposit"
--> not sure we should specify it concerns checks, cash etc...